packages/algob/test/stubs/algo-operator.ts

```diff
@@ -1,12 +1,18 @@
 
-import { Algodv2 } from "algosdk";
+import algosdk, { Algodv2 } from "algosdk";
 
 import { AlgoOperator } from "../../src/lib/algo-operator";
-import { Account, ASADef, ASADeploymentFlags, ASAInfo } from "../../src/types";
+import {
+  Account, ASADef, ASADeploymentFlags, ASAInfo, ASCCache, ASCDeploymentFlags,
+  ASCInfo
+} from "../../src/types";
 
 export class AlgoOperatorDryRunImpl implements AlgoOperator {
   get algodClient (): Algodv2 {
-    throw new Error("Not implemented");
+    const token = "adsadasd";
+    const host = "127.0.0.1";
+    const port = 42981;
+    return new algosdk.Algodv2(token, host, port);
```

Comment:
Remove this config. Throw exception or mock the methods using `sinon` library.
What uses this? This shouldn't be used from Deployer. It's only provided for the user. You have your `CompileOp` which takes care of `Algodv2` and Deployer must not use it directly.